### ISSUE TITLE
Redesign the Bassa Landing page

### ISSUE THREAD

**Issue by rehrumesh** [scorelab/Bassa#67]

**Comment by yruwanmalee** [comment]
The positioning of the input fields and the buttons need changes so it will actually make it easier for the user. Furthermore, the area of login form needs to be emphasized which can be done by changing the background color with regard to the whole screen.

**Comment by rehrumesh** [comment]
@yruwanmalee any mockups?

**Comment by saisankargochhayat** [comment]
I think , the white background goes well with the app.

**Comment by yruwanmalee** [comment]
![bassa5](https://cloud.githubusercontent.com/assets/14134332/24070077/ec4c6e06-0bdb-11e7-87af-ab0121b16b76.png)

This is what I have been working lately.

**Comment by yruwanmalee** [comment]
Well the color change would not be something bright but thought of making a subtle difference of the colors between login form compared to the whole screen.

**Comment by yruwanmalee** [comment]
@rehrumesh  why is not the name of the user not included in the signup form?

**Comment by rehrumesh** [comment]
@yruwanmalee What do you mean?

**Comment by rehrumesh** [comment]
@yruwanmalee appriciate the effort with the login page work. But I feel like the consistency between button styles has been lost. Any opinions?

**Comment by malithsen** [comment]
@rehrumesh agreed. +1 for vertically stacked inputs as opposed to horizontal ones we have now @yruwanmalee

**Comment by yruwanmalee** [comment]
@rehrumesh Don't we need to record the user's first name and last name in the signup form?

**Comment by rajikaimal** [comment]
How about an anchor instead of a button for signup. Next to 'don't have an account', `signup` can be an anchor tag. Just a suggestion.

**Comment by yruwanmalee** [comment]
@rehrumesh I think this way the button consistency issue will be resolved.
![bassa6](https://cloud.githubusercontent.com/assets/14134332/24209061/3687f0fe-0f4b-11e7-8880-46bd0c07563c.png)

**Comment by rehrumesh** [comment]
@yruwanmalee what do we have near "Don't have an account"?

**Comment by yruwanmalee** [comment]
@rehrumesh It was used to navigate an individual for the signup page. Anyway, the UI was changed theresince. I have also done some customizations to the alerts as well. Both of it can be seen in the screenshot I have attached below.
![abcc](https://cloud.githubusercontent.com/assets/14134332/24648130/39297090-1940-11e7-81b3-5a1e760cc590.png)

**Comment by yruwanmalee** [comment]
@rehrumesh  are there any modifications i have got to do? :)

**Comment by Paarmita** [comment]
@yruwanmalee I guess the signup should also be highlighted by making it button/ underline it.
